gcalbot/gcalbot/event.go

```diff
@@ -99,3 +100,38 @@ func FormatEvent(
 	return fmt.Sprintf(message,
 		url, what, when, where, conferenceData, organizer, calendarSummary, description), nil
 }
+
+func FormatEventSchedule(
+	events []*calendar.Event,
+	timezone *time.Location,
+	format24HourTime bool,
+) (schedule string, err error) {
+	// don't know the length of the array since we skip all-day events
+	sort.Slice(events, func(i, j int) bool {
+		startLeft, _, _, _ := ParseTime(events[i].Start, events[i].End)
```

Comment:
bit of a bummer to have to reparse on every comparison, esp since we ignore the end time

Reply:
Yeah I was considering storing the data in an array or something, do you think that would be worth it?

Reply:
yeah it won't be too bad, you can filter out all day events when building that array too